CMakeLists.txt

```diff
@@ -230,6 +230,11 @@ Enable this if the module uses typeid or dynamic_cast. You will probably need to
         option(PCH "Whether to use precompiled headers" ON)
     endif()
 
+    option(ENABLE_HELP "Enable Help" ON)
+    if(ENABLE_HELP)
+        add_definitions(-DENABLE_HELP)
+    endif()
```

Comment:
This looks a bit too pervasive in the build system for now.

Reply:
Used `target_compile_definitions` instead of global `add_definitions`.